### PR TITLE
Add start and end dates to create viewing room mutation (GALL-2746)

### PR DESCRIPTION
**Pull request by iskounen**
This PR adds the date fields (`startAt`, `endAt`, and `timeZone`) to the input type for the `createViewingRoom` mutation.

The mutation to test once https://github.com/artsy/gravity/pull/13115 is merged:

```
mutation {
  createViewingRoom(input: {partnerId: "5141c8b2bf12487ca00003db", title: "Foo", startAt: "2020-05-29T00:00:00", endAt: "2020-06-29T00:00:00", timeZone: "America/New_York", introStatement: "Allow myself to introduce... myself.", pullQuote: "You can observe a lot by just watching.", body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed nisl lorem, placerat non rhoncus eu, sodales id eros. Aliquam urna metus, laoreet eget lobortis rhoncus, sodales blandit elit. Vestibulum mattis porta nulla, sit amet sodales eros consequat sit amet. Mauris eu odio sed sapien lobortis tristique. Etiam mollis enim sed vehicula elementum. Ut auctor ornare tortor, at gravida dolor cursus non. Mauris placerat elit libero, quis vestibulum lacus placerat eu. Maecenas id convallis odio. Sed vulputate ultrices ex, non elementum justo auctor non. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu libero interdum, congue dui vitae, mattis nibh."}) {
    viewingRoomOrErrors {
      ... on ViewingRoom {
        title
        startAt
        endAt
        timeZone
        introStatement
        pullQuote
        body
      }
    }
  }
}
```

### CODE REVIEW

File: _schema.graphql [hunk 1]
```diff
@@ -5660,6 +5669,7 @@ input EnableSecondFactorInput {
 type EnableSecondFactorPayload {
   # A unique identifier for the client performing the mutation.
   clientMutationId: String
+  recoveryCodes: [String!]
```
Comment (iskounen) [inline _schema.graphql:5672]: @dblandin I wanted to double-check that this was OK to add this to the Metaphysics schema.

File: _schema.graphql [hunk 2]
```diff
@@ -12935,11 +12945,15 @@ type ViewingRoom {
 
   # ID of the partner associated with this viewing room
   partnerID: String!
+  published: Boolean!
```
Comment (iskounen) [inline _schema.graphql:12948]: @oxaudo and @mdole I wanted to double-check that this was OK to add this to the Metaphysics schema.

File: peril/compareForceSchema.ts [hunk 3]
```diff
@@ -4,31 +4,28 @@ import fetch from "node-fetch"
...
-// and the current Reaction one, warn.
+// and the current Force one, warn.
 export default async () => {
-  const forcePackageJSON = await (await fetch(
-    "https://raw.githubusercontent.com/artsy/force/master/package.json"
-  )).json()
-  const reactionVersion = forcePackageJSON["dependencies"]["@artsy/reaction"]
-  const reactionSchemaUrl = `https://github.com/artsy/reaction/raw/v${reactionVersion}/data/schema.graphql`
+  const forceSchemaUrl =
+    "https://github.com/artsy/force/blob/master/data/schema.graphql"
```
Comment (ds300) [inline peril/compareForceSchema.ts:10]:
```suggestion
    "https://github.com/artsy/force/raw/master/data/schema.graphql"
```

Comment (iskounen) [reply peril/compareForceSchema.ts]: Thanks a lot for checking that out for me. I'm testing it against Danger again....

Comment (iskounen) [reply peril/compareForceSchema.ts]: ... and it's not going away 😬

**Pull request merged**